### ISSUE TITLE
I wanted to generate 500 products, got 5k after aborted the generation process

### ISSUE THREAD

**Issue by RomainMazB** [oc-shopaholic/oc-fake-data-shopaholic-plugin#11]
Hi,
I faced this issue on a fresh install: I just wanted to generate 500 products, using the CLI:
![image](https://user-images.githubusercontent.com/53976837/84577056-5bdd3480-adb9-11ea-8e7e-e9eacce9c236.png)

After one or two minutes I understood something wrong was happening, so I've aborted the generation process, looked at the products list in the backend: more than 5k of products was generated:
![image](https://user-images.githubusercontent.com/53976837/84577063-65ff3300-adb9-11ea-90f5-b7522b81baca.png)

Don't know what happened, but it should be easy to reproduce.

Maz :)

**Comment by kharanenka** [comment]
Hi! Each set contains a certain amount of products. For example, from 15 to 21. In CLI you specify how many repetitions of generations will be performed. For example, 500 repetitions * 21 products in one cycle = 10500.

**Comment by lautsevich** [comment]
@kharanenka, we should improve CLI UX in this case. I believe it would be better te set the final quantity of the products in the catalog.

**Comment by RomainMazB** [comment]
I agree with @lautsevich ,
Even if the question asked in cli give you the point: "how many times to repeat the creating of product?", the new user I am understand "how many products do you want?"

I understand maybe you need this small lists of products for randomness/convenience reason, I don't want all my product to be called nike-1 nike-2, nike-3 and so on.

But I'd prefer if the loop just stopped when I had at least 500 products:
21products, executed 24 times (21*24=504)

**Comment by lautsevich** [comment]
@RomainMazB, yep, it would be better not to add postfix number. It looks so unnatural.
